### DISCUSSION TITLE
Fragen zur Vorlesung TD-IDF - Zipfs Gesetz

### DISCUSSION THREAD

**Discussion by ghost** [irgroup/dis12-2020#2] [Q&A] [answered]
Hallo Herr Schaer,

zu Zipfs Gesetz habe ich noch folgende Fragen:

- Was passiert bei der zweiten Umwandlung der Formel zu log(𝑓t) = log 𝑘 − log 𝑟t)? (Folie 23 im Skript)

- Auf Folie 25 heißt es "Die aussagekräftigen Begriffe stehen nicht in jedem Dokument!" - die Logik des Satzes ist mir nicht klar. Heißt das soviel wie "Die aussagekräftigen Begriffe stehen nur in den relevanten Dokumenten" oder sollte es eigentlich heißen, "nicht alle bzw. die meisten Begriffe im Dokument sind nicht aussagekräftig"?

Können Sie das am Freitag noch einmal erläutern, bitte?

Vielen Dank und bis dann
Helga Wolf

**Comment by phschaer** [comment, marked as answer]
Liebe Frau Wolf,

- Die zweite Umwandlung ist nur die Anwendung einer Logarithmus-Regel. Kann man z.B. hier nachlesen: https://www.frustfrei-lernen.de/mathematik/logarithmus-regeln.html

- Uff, die erste Interpretation würde ich so nicht mitgehen, die zweite aber schon. 

Aber ja, das besprechen wir sehr gerne morgen! 

Schönen Abend

Philipp Schaer

**Comment by ghost** [comment]
danke :-)
